app/cells/attendee/form_cell.rb

```diff
@@ -70,6 +71,15 @@ def contact_inputs
     end
   end
 
+  def rec_center_pass_inputs
+    inputs 'Rec Center Pass' do
+      datepicker_input(model, :rec_center_pass_started_at,
+                       label: 'Rec Center Pass Start Date')
```

Comment:
Instead of specifying the label names here, consider adding them to [config/locales/en.yml](https://github.com/CruGlobal/staff-conf-onsite/blob/master/config/locales/en.yml).